File: templates/androidstarters-java/app/src/main/java/io/mvpstarter/sample/features/detail/DetailMvpView.java
package <%= appPackage %>.features.detail;

import <%= appPackage %>.data.model.response.Pokemon;
import <%= appPackage %>.data.model.response.Statistic;
import <%= appPackage %>.features.base.MvpView;

public interface DetailMvpView extends MvpView {

    void showPokemon(Pokemon pokemon);

    void showStat(Statistic statistic);

    void showProgress(boolean show);

    void showError(Throwable error);
}

File: templates/androidstarters-kotlin/app/src/main/java/io/mvpstarter/sample/features/detail/DetailMvpView.kt
package <%= appPackage %>.features.detail

import <%= appPackage %>.data.model.Pokemon
import <%= appPackage %>.data.model.Statistic
import <%= appPackage %>.features.base.MvpView

interface DetailMvpView : MvpView {

    fun showPokemon(pokemon: Pokemon)

    fun showStat(statistic: Statistic)

    fun showProgress(show: Boolean)

    fun showError(error: Throwable)

}